.github/workflows/test.yml

```diff
@@ -10,30 +10,22 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.8
-
-    - name: Install dependencies
-      run: python -m pip install black tox
-
-    - name: "Run black"
-      run: black --check .
-
+    - name: Extract branch name
+      shell: bash
+      run: echo "##[set-output name=tag;]$(echo ${GITHUB_REF#refs/heads/} | tr / -)"
+      id: extract_branch
 
-    - name: "Cache docker build"
-      uses: actions/cache@v2
-      id: cache-docker
-      with:
-        path: '/var/lib/docker/'
-        key: 'constant-because-docker-handles-cache'
-
-    - name: "Pull marine-core docker image"
-      run: docker pull domerd/marine-core:marine
+    - name: "Pull core image"
+      shell: bash
+      run: echo -n "##[set-output name=tag;]" && docker pull domerd/marine-core:${{ steps.extract_branch.outputs.tag }} > /dev/null && echo "${{ steps.extract_branch.outputs.tag }}" || echo marine
```

Comment:
If it's not difficult let's fix the naming convention of the repos

Reply:
I think that's up to you - open a dockerhub user and update the credentials.